src/components/TextButton/docs/index.story.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { TEXT_BUTTON_PRIORITY, TextButton } from '../';
+import Markdown from 'wix-storybook-utils/Markdown';
+import {
+  importExample as importSourceExample,
+  StyledExample,
+} from './examples';
+import {
+  header,
+  api,
+  divider,
+  importExample,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+import { allComponents } from '../../../../stories/utils/allComponents';
+import * as Readme from '../README.md';
+
+const code = config =>
+  baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+  category: 'Components',
+  storyName: 'TextButton',
+  component: TextButton,
+  componentPath: '../TextButton.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-TextButton',
+    children: 'Text Button',
+    priority: TEXT_BUTTON_PRIORITY.link,
+  }),
+  exampleProps: {
+    priority: Object.values(TEXT_BUTTON_PRIORITY),
+  },
+  dataHook: 'storybook-TextButton',
+  sections: [
+    header(),
+    tabs([
+      tab({
+        title: 'Usage',
+        sections: [
+          <Markdown key="readme" source={Readme} />,
+
+          importExample({
+            source: importSourceExample,
+          }),
+
+          divider(),
+
+          title('Examples'),
+        ],
+      }),
+
+      ...[
+        { title: 'API', sections: [api()] },
+        { title: 'TestKit', sections: [testkit()] },
+        { title: 'Playground', sections: [playground()] },
+      ].map(tab),
+    ]),
+  ],
+  examples: <StyledExample />,
```

Comment:
For some reason the examples aren't rendered, you can see here:
https://wix-wix-ui-tpa-pr-125.surge.sh/?path=/story/components--textbutton

Maybe it's because of the `Markdown` component.
You can compare it to the `Pagination` storybook

Reply:
fixed